docs/zh_cn/advanced_guides/datasets.md

```diff
@@ -9,9 +9,10 @@
 实例化 Cityscapes 训练数据集:
 
 ```python
+from mmengine.registry import init_default_scope
 from mmseg.datasets import CityscapesDataset
-from mmseg.utils import register_all_modules
-register_all_modules()
+
+init_default_scope('mmseg')
```

Comment:
same problem as above

Reply:
> same problem as above

Here are example codes for initializing a dataset, I think this line should not be removed.